sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/ManifestFileCommitProtocol.scala

```diff
@@ -70,7 +80,24 @@ class ManifestFileCommitProtocol(jobId: String, path: String)
 
   override def abortJob(jobContext: JobContext): Unit = {
     require(fileLog != null, "setupManifestOptions must be called before this function")
-    // Do nothing
+    // best effort cleanup of complete files from failed job
+    // since the file has UUID in its filename, we are safe to try deleting them
+    // not deleting them doesn't help reducing cost to saving files when retrying
```

Comment:
If this is a new sentence, please start with upper case. BTW, could you rewrite this line more simply? :)

Reply:
Yeah thanks for the suggestion. The comment looks like trying to explain necessity of the lines which seems not needed. I'll refine the sentences a bit.